modules/oadp-timeouts.adoc

```diff
@@ -0,0 +1,194 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-timeouts_{context}"]
+= OADP timeouts
+
+Extending a timeout allows complex or resource-intensive processes to complete successfully without premature termination, reducing the likelihood of errors, retries or failures. However, it is necessary to strike a balance when extending timeouts. Setting excessively long timeouts may hide underlying issues in the process. Therefore, careful consideration and monitoring are necessary to determine an appropriate timeout value that meets the needs of the process and the overall system performance.
+
+Below are various OADP timeouts, with instructions of how and when to implement these parameters:
+
+[id="Restic-timeout_{context}"]
+== Restic timeout
+
+.Description
+`timeout` defines the Restic timeout. The default value is 1h.
+
+.Implementation
+.. Use this parameter for Restic backups with large volumes.
```

Comment:
```suggestion
.. Use this parameter for Restic backups that contain large storage volumes.
```

Reply:
What constitutes a _large_ volume? Is there a hard metric to consider?

Reply:
Large - total PV data usage >500GB